dubbo-go/dubbo-go.go

```diff
@@ -0,0 +1,204 @@
+//
+// Copyright 2021 SkyAPM org
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package dubbo_go
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"time"
+
+	"dubbo.apache.org/dubbo-go/v3/common/extension"
+	"dubbo.apache.org/dubbo-go/v3/filter"
+	"dubbo.apache.org/dubbo-go/v3/protocol"
+	agentv3 "skywalking.apache.org/repo/goapi/collect/language/agent/v3"
+
+	"github.com/SkyAPM/go2sky"
+)
+
+const (
+	// need add to component library
+	componentIDGo2SkyClient = 5013
+	componentIDGo2SkyServer = 5014
```

Comment:
Take a look at this, https://github.com/apache/skywalking/blob/ebd040eb1a3ed92e79da6f917cb9c9fe44a6938d/oap-server/server-starter/src/main/resources/component-libraries.yml#L41-L43.

You don't need client and server, also, you don't need new component IDs. You could update `languages` field in the YAML of main repo.

Reply:
Mapping is for indicating a server side which is not tracing. Such as MySQL JDBC driver to MySQL server. For MySQL server, we don't expect an agent installed.